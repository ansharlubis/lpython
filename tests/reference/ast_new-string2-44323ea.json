{
    "basename": "ast_new-string2-44323ea",
    "cmd": "lpython --show-ast --new-parser --no-color {infile} -o {outfile}",
    "infile": "tests/parser/string2.py",
    "infile_hash": "fc6c3d133d5b93c5b5f052b9d86c2b1f270149aa48dba8ae8ced3e86",
    "outfile": null,
    "outfile_hash": null,
    "stdout": "ast_new-string2-44323ea.stdout",
    "stdout_hash": "cf514f21d4a57de66c5fbfc107a6ab58818a09359ef9ba651896d71f",
    "stderr": null,
    "stderr_hash": null,
    "returncode": 0
}