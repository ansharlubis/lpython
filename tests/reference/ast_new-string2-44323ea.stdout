(Module [(Expr (ConstantStr "\"\\" ())) (Expr (ConstantStr "
\\" ())) (Expr (ConstantStr "\\" ())) (Expr (ConstantStr "   \\" ())) (Expr (ConstantStr "\\ " ())) (Expr (ConstantStr "\
" ())) (Expr (ConstantStr "\\|" ())) (Expr (ConstantStr "Text\\" ())) (Expr (ConstantStr "\
" ())) (Expr (Call (Name x Load) [(ConstantStr "\
class a:
    def test():
        return r"\\LaTeX"
" ())] []))] [])
